week-1/articles/installfest.md

```diff
@@ -39,12 +51,7 @@ You will also be show how to sign up for a user account on the following service
 2. Click on the link for Windows
 3. Run the installer from your downloads folder
 4. Select the default options
-
-#### Github User Account
-
-1. Visit the Github Signup page at <https://github.com/join>
-2. Create a user account and select a unique user name.
```

Comment:
Unique but professional. I would definitely advise them that it should not be pokemon_maniac66@aol.com (I've had that issue in the past). It should be something you wouldn't be afraid to show your grandma.